sig-testing/charter.md

```diff
@@ -0,0 +1,82 @@
+# SIG Testing Charter
+
+This charter adheres to the conventions described in the 
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+SIG Testing is interested in effective testing of Kubernetes.  We do not write or
+troubleshoot the project's tests, but instead focus on tooling that makes it
+easier for the community to write and run tests, and to contribute, analyze and
+act upon test results.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Project CI and merge automation via tools such as [prow] and [tide]
+- Extraction of test results from GCS and populating a public accessible
+  BigQuery dataset via [kettle]
+- Display and analysis of test artifacts via tools like [gubernator], 
+  [testgrid], [triage] and [velodrome]
+- Configuration management of jobs and ensuring they use a consistent
+  process via tools such as [job configs], [kubetest]
+- Tools that facilitate local testing of kubernetes such as [greenhouse] 
+  and [kind]
+- Jobs that automate away project toil via [@fejta-bot]
+- Ensuring all of the above is kept running on a best effort basis
+
+#### Cross-cutting and Externally Facing Processes
+
+- The Release Team [test-infra role] is staffed by a memer of SIG Testing, as
+  such their responsibilities are within the scope of this SIG, including
+  the maintenance of release jobs
+- When rolling out changes that may potentially impact the project as a whole
+  we consult with SIG Contributor Experience, and follow [lazy consensus] by 
+  notifying kubernetes-dev, providing a deadline,and a rationale for the 
```

Comment:
For discussion: this is really squishy, how can we firm this up

Reply:
Seems accurate to the process we use, though -- do you mean we should firm up the process or this doc?